src/components/widgets/accounts-cash-balance/accounts-cash-balance.less

```diff
@@ -4,12 +4,12 @@
   .data-container {
     height: ~"calc(@{impac-big-widget-size} - 20px)";
     width: 100%;
-    display: flex;
   }
 
   .left-panel {
-    flex: 0;
-    min-width: 180px;
+    display: inline-block;
+    width: 180px;
+    height: ~"calc(@{impac-big-widget-size} - 20px)";
```

Comment:
I came up with the exact same solution in my current branch :)
I don't think you need this line though (height: ~"calc(@{impac-big-widget-size} - 20px)";) as the height is already set by .cash-balance-chart?